reference/forms/types/dateinterval.rst

```diff
@@ -0,0 +1,325 @@
+.. index::
+   single: Forms; Fields; DateIntervalType
+
+DateIntervalType Field Type
+===========================
+
+.. versionadded:: 3.2
+   The DateIntervalType field type was introduced in Symfony 3.2.
+
+This field type allows the user to modify data that represents a specific
+date interval. It can be rendered as a integer or text input or select tags.
+
+The underlying format of the data can be a ``DateInterval`` object,
+a `ISO 8601`_ duration string (e.g. ``P1DT12H``) or an array.
+
++----------------------+-----------------------------------------------------------------------------------+
+| Underlying Data Type | can be ``DateInterval``, string or array (see the ``input`` option)               |
++----------------------+-----------------------------------------------------------------------------------+
+| Rendered as          | single text box or up to six select, text or integer boxes plus optional checkbox |
```

Comment:
single text box, multiple text boxes or select fields - see the `widget`_ option